Package.pins

```diff
@@ -1,42 +0,0 @@
-{
```

Comment:
Removed this. Don't think we were using pins anymore (given the old deps listed).